### ISSUE TITLE
Yahoo JSON not updating

### ISSUE THREAD

**Issue by davidamaral1005** [finance-quote/finance-quote#271]
![Screenshot (119)](https://github.com/finance-quote/finance-quote/assets/133290746/50ad8403-57d9-46e1-afde-59aff47f2cbb)

https://query1.finance.yahoo.com/v7/finance/quote?symbols=LLY
gets response of:
{"finance":{"result":null,"error":{"code":"Unauthorized","description":"Invalid Crumb"}}}

**Comment by bpschuck** [comment]
Duplicate of an issue already addressed in PR #268.

**Comment by davidamaral1005** [comment]
Thank you. made the workaround change and all is well with the world.
    On Friday, May 12, 2023 at 07:13:50 AM GMT-7, Bruce P Schuck ***@***.***> wrote:  
 
 


Duplicate of an issue already addressed in PR #268.

**Comment by bpschuck** [comment]
New URL in v1.55 release pushed to CPAN today.

**Comment by davidamaral1005** [comment]
This started again. same issue and i have yahoojson.pm set to version 6
Finance::Quote: 1.55

    On Saturday, May 13, 2023 at 01:37:19 PM GMT-7, Bruce P Schuck ***@***.***> wrote:  
 
 


Closed #271 as completed.

**Comment by bpschuck** [comment]
Looks okay to me:

```
$ curl 'https://query1.finance.yahoo.com/v6/finance/quote?symbols=F' | jq  
{
  "quoteResponse": {
    "result": [
      {
        "language": "en-US",
        "region": "US",
        "quoteType": "EQUITY",
        "typeDisp": "Equity",
        "quoteSourceName": "Nasdaq Real Time Price",
        "triggerable": true,
        "customPriceAlertConfidence": "HIGH",
        "currency": "USD",
        "exchangeTimezoneName": "America/New_York",
        "exchangeTimezoneShortName": "EDT",
        "gmtOffSetMilliseconds": -14400000,
        "marketState": "PREPRE",
        "regularMarketChangePercent": 0.85984033,
        "regularMarketPrice": 11.73,
        "shortName": "Ford Motor Company",
        "longName": "Ford Motor Company",
        "messageBoardId": "finmb_106335",
        "market": "us_market",
        "esgPopulated": false,
        "exchange": "NYQ",
        "postMarketChangePercent": 0.341014,
        "postMarketTime": 1684886394,
        "postMarketPrice": 11.77,
        "postMarketChange": 0.0400009,
        "regularMarketChange": 0.09999943,
        "regularMarketTime": 1684872187,
        "regularMarketDayHigh": 11.91,
        "regularMarketDayRange": "11.53 - 11.91",
        "regularMarketDayLow": 11.53,
        "regularMarketVolume": 55780207,
        "regularMarketPreviousClose": 11.63,
        "bid": 0,
        "ask": 0,
        "bidSize": 400,
        "askSize": 40,
        "fullExchangeName": "NYSE",
        "financialCurrency": "USD",
        "regularMarketOpen": 11.53,
        "averageDailyVolume3Month": 65054547,
        "averageDailyVolume10Day": 49694790,
        "fiftyTwoWeekLowChange": 1.1199999,
        "fiftyTwoWeekLowChangePercent": 0.10556079,
        "fiftyTwoWeekRange": "10.61 - 16.68",
        "fiftyTwoWeekHighChange": -4.950001,
        "fiftyTwoWeekHighChangePercent": -0.29676262,
        "fiftyTwoWeekLow": 10.61,
        "fiftyTwoWeekHigh": 16.68,
        "dividendDate": 1685577600,
        "earningsTimestamp": 1683061200,
        "earningsTimestampStart": 1690315500,
        "earningsTimestampEnd": 1690833900,
        "trailingAnnualDividendRate": 0.55,
        "trailingPE": 16.068493,
        "trailingAnnualDividendYield": 0.047291487,
        "epsTrailingTwelveMonths": 0.73,
        "epsForward": 1.65,
        "epsCurrentYear": 1.77,
        "priceEpsCurrentYear": 6.6271186,
        "sharesOutstanding": 3929920000,
        "bookValue": 10.589,
        "fiftyDayAverage": 11.947,
        "fiftyDayAverageChange": -0.21700001,
        "fiftyDayAverageChangePercent": -0.018163556,
        "twoHundredDayAverage": 12.96445,
        "twoHundredDayAverageChange": -1.2344503,
        "twoHundredDayAverageChangePercent": -0.09521811,
        "marketCap": 46929031168,
        "forwardPE": 7.109091,
        "priceToBook": 1.1077533,
        "sourceInterval": 15,
        "exchangeDataDelayedBy": 0,
        "averageAnalystRating": "2.9 - Hold",
        "tradeable": false,
        "cryptoTradeable": false,
        "firstTradeDateMilliseconds": 76253400000,
        "priceHint": 2,
        "symbol": "F"
      }
    ],
    "error": null
  }
}

$ perl -MFinance::Quote -e 'print "$Finance::Quote::VERSION\n"'
1.55
```
If Yahoo does disable the v6 API, I would suggest switching to AlphaVantage.
